pysat/_instrument.py

```diff
@@ -307,10 +307,10 @@ def __init__(self, platform=None, name=None, tag=None, inst_id=None,
             self.multi_file_day = multi_file_day
 
         # Initialize the padding
-        if isinstance(pad, pds.DateOffset) or pad is None:
+        if isinstance(pad, dt.timedelta) or pad is None:
             self.pad = pad
         elif isinstance(pad, dict):
-            self.pad = pds.DateOffset(**pad)
+            self.pad = dt.timedelta(**pad)
         else:
             raise ValueError(''.join(['pad must be a dict, NoneType, or ',
                                       'pandas.DateOffset instance.']))
```

Comment:
This would need to be updated.

Reply:
I'm not sure if we want to change to datetime here.  We would lose the possibility of padding data by months or years (potetnially useful in simple datasets like f107?).  Thoughts?

Reply:
I think that padding is useful and warrants keeping pds.DateOffset here.  Should we allow timedelta as well, though?

Reply:
How about we allow both, and if it's a dict, assume pds.DateOffset to interpret, since it is more robust?

Reply:
> assume pds.DateOffset to interpret

We may need an additional test to enforce this, but yes.

Reply:
updated the test using parametrize to ensure that both options work